### PR TITLE
feat(DIST-1904): Max width for slider

### PR DESCRIPTION
**Pull request by scarciofolomarco**

### CODE REVIEW

**Comment by typeform-ops-gha** [comment]
*[BOT]* Preview available with hash `4765be7ce0734402f64db147bb07be64df4cdab0` [here](https://embed.typeform.com/4765be7ce0734402f64db147bb07be64df4cdab0/embed.js).

File: packages/embed/src/factories/create-slider/create-slider.ts [hunk 1]
```diff
@@ -18,7 +18,9 @@ import { SliderOptions } from './slider-options'
 
 export type Slider = EmbedPopup
 
-const buildSlider = (position: 'right' | 'left') => {
+type Position = 'right' | 'left'
```
Comment (mathio) [inline packages/embed/src/factories/create-slider/create-slider.ts:21]: Nice 🚀

**Review by mathio — APPROVED** [review]

**Comment by typeform-ops-gha** [comment]
*[BOT]* Preview available with hash `02cce6ea3a54fa82f9de8024771fbaace5b785c8` [here](https://embed.typeform.com/02cce6ea3a54fa82f9de8024771fbaace5b785c8/embed.js).

**Pull request merged**